backend/README.md

```diff
@@ -1,5 +1,23 @@
 ## Local Installation
 1) clone the repo
 2) enter the repo
-3) run `docker-compose up`
-4) exit using Ctrl-C or `docker-compose down`
+3) run `docker-compose up -d`
+4) `docker ps`, note container ID of backend_web image
+5) `docker exec -it [CONTAINER_ID] bash`
+7) `python manage.py migrate`
+8) exit using Ctrl-C or `docker-compose down`
+
+- This is super hacky because you have to run migrations manually
+- The alternative is to build out some logic where the docker web container waits for the docker db container to be ready
```

Comment:
Look at this Dockerfile and .docker/entrypoint.sh on how to handle this, i think it'll make our lives a lot easier
https://github.com/CS130-W20/team-B1/tree/master/backend